src/engine/sparqlExpressions/ConvertToNumericExpression.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright 2024
+
+#include "engine/sparqlExpressions/NaryExpressionImpl.h"
+
+namespace sparqlExpression {
+namespace detail::to_numeric {
+
+// class that converts an input `int64_t`, `double` or `std::string`
+// to a numeric value `int64_t` or `double`
+template <typename T>
+class ToNumericImpl {
+ private:
+  Id getFromString(std::string& input) const {
```

Comment:
argument can be const&, doesn't sonarcloud complain here?

Reply:
Seems like it doesn't complain